### ISSUE TITLE
Feature request: Can you add option to filter out themes from extension's list?

### ISSUE THREAD

**Issue by vhanla** [Microsoft/vscode#49013]
- VSCode Version: 1.22.2
- OS Version: Windows 10 x64

Steps to Reproduce:

1. Access extensions sidebar
2. Try to figure out which one is a theme

Feature request: Add option to filter out themes on extension's sidebar.

As part of customization of this awesome code editor, we try as many themes available in order to find out the one that fits our taste, eventually we end up with lots of them installed. However, it becomes hard to uninstall the ones we don't like since it is listed among other extensions. And checking each one to find out if that is a theme, takes time, since it has to load its info from internet. There is a HotKey to switch among themes, and it is helpful, since it lists themes by its name, so one can know a specific theme's name, in order to search in extensions list, but that takes time too.

The same occurs the other way round, when we want to tidy up our installed extensions, and uninstall those we don't need anymore, but themes gets in our way, making us spend more time in case we don't want to remove any theme, but only proper extensions.

In conclusion. It would be nice to filter out @themes be it for showing them or hiding them from the extension's list. 
I know this is not a priority, but it would be very helpful.

**Comment by sandy081** [comment]
You can use the filter `category:themes` to search for themes extensions.
